### PR TITLE
Gray out editors

### PR DESCRIPTION
**Pull request by William-Hill**
Addresses issue #323

### CODE REVIEW

**Comment by doutriaux1** [comment]
@William-Hill awesome! Thx!

**Comment by downiec** [comment]
It looks like some tests failed. I noticed this: TypeError: Cannot read property 'replace' of undefined which shows up in the AddEditRemoveNav.jsx file. I think when the title string is empty/undefined, the '.replace' method call causes an error. I think if that's fixed it should pass the tests.

File: frontend/src/js/components/AddEditRemoveNav/AddEditRemoveNav.jsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ class AddEditRemoveNav extends PureComponent {
                                 </a>
                             </li>
                             <li className="action-edit-button">
-                                <a href="#" onClick={this.props.editAction} title={this.props.editText ? this.props.editText : ""}>
-                                    <i className={`glyphicon glyphicon-edit ${this.props.editAction ? "" : "disabled"}`}></i>
+                                <a href="#" id={`${this.props.title.replace(" ", "-")}-anchor`} onClick={this.props.editAction} title={this.props.editText ? this.props.editText : ""}>
+                                    <i id={`${this.props.title.replace(" ", "-")}`} className={`glyphicon glyphicon-edit ${this.props.editAction ? "" : "disabled"}`}></i>
```
[the remark this answers is not in the document]
Comment (downiec) [reply frontend/src/js/components/AddEditRemoveNav/AddEditRemoveNav.jsx]: Yeah I agree a default value would be a good solution for this case. I mentioned that to William yesterday, not sure if he's had a change to look at it yet though.

[the remark this answers is not in the document]
Comment (William-Hill) [reply frontend/src/js/components/AddEditRemoveNav/AddEditRemoveNav.jsx]: Added the default title as suggested by @downiec and @sterlingbaldwin

**Comment by sterlingbaldwin** [comment]
Looks good with the default title.

**Pull request merged**